shared/utils/urls.ts

```diff
@@ -86,12 +87,14 @@ export function sanitizeUrl(url: string | null | undefined) {
     return undefined;
   }
 
+  const protocol = url.match(/(\w+):\/\//);
+
   if (
-    !isUrl(url) &&
-    !url.startsWith("/") &&
-    !url.startsWith("#") &&
-    !url.startsWith("mailto:") &&
-    !url.includes("://")
+    (!isUrl(url) &&
+      !url.startsWith("/") &&
+      !url.startsWith("#") &&
+      !url.startsWith("mailto:")) ||
+    (protocol && blockedProtocols.includes(protocol[0]))
```

Comment:
I think we can stick with using `startsWith("javascript://")` pattern here and inline the protocols we care about, it'll be more maintainable/simple that way and the regex above is too loose anyway

Reply:
Changes have been committed @tommoor, let me know if you want me to squash the commits